### PR TITLE
Math magicians: setup

### PR DESCRIPTION
**Pull request by rubydevi**
**Description:**
I have set up the environment and tools needed to develop a React application. In the following projects, I will develop the actual application.

**Changes Made:**

- Installed Linters for React and Redux
- Updated README.md
- Followed Gitflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,166 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (V-Blaze) [inline README.md]: - [ ] You have done great so far, your react app was set up correctly. :+1: :+1: But there are some changes you need to make to your readme before moving to the next milestone. :100: 
- [ ] Currently, this issue link doesn't lead to the issue page of this repository, it leads to a 404 error page. I'll suggest you fix this as broken links in your readme makes it look a bit unprofessional. :+1: 

- - Screenshot for reference:
 
Screenshot from 2023-07-10 13-57-00

File: README.md [hunk 2]
```diff
@@ -1,70 +1,166 @@
...
+## Acknowledgments <a name="acknowledgements"></a>
+
+- I'd like to thank Microverse for this wonderful README template.
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (V-Blaze) [inline README.md]: - [ ] This license link also leads to a 404 error page. I'll suggest you add a license file to your project and link it appropriately. :smile: :wave:   
- - Screenshot:
 
Screenshot from 2023-07-10 13-57-34

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @rubydevi , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Used **Create react app**✔
-  Linters setup correctly👋 
- Correct git flow ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @rubydevi,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**